Add profile page

EvgeniiMaiorov opened the pull request:
Where?
https://trello.com/c/lUg0rI0h/10-add-user-profile-page

What?
Create a profile page to display user data

How?
Add profile page and basic logic

Commit:
Add profile page

Where?
https://trello.com/c/lUg0rI0h/10-add-user-profile-page

What?
Create a profile page to display user data

How?
Add profile page and basic logic

anevsevra commented on app/javascript/packs/components/ProfilePage.jsx:
```diff
@@ -0,0 +1,172 @@
+  const [user, setUser] = useState({})
+
+  useEffect(() => {
+    axios.get('/api/v1/current_user', { headers: { Authorization: props.userToken } })
```
Use `/api/v1/users/:id` instead. ID can be fetched from token.

Commit:
Remove current_user and use react-jwt lib

Where?
https://trello.com/c/PrKic2WU/27-remove-currentuser-and-use-react-jwt-lib

What?
Add a react-jwt library to decode the token
Refuse to use the current user method

How?
Add a react-jwt library to decode the token
Refuse to use the current user method
Update routes
Update frontend logic
